scripts/createDataJson.js

```diff
@@ -0,0 +1,53 @@
+const program = require('commander');
```

Comment:
I don't see this new module in package.json dependecies

Reply:
It is in node modules, no need to add as a new package

Reply:
Well, it's a good practice to do so if we are using it directly. Its likely a peerDependency of another module. NPM installs it but we should have it listed in dev dependencies as well.

Reply:
Cool! will do it, it is a dependency of formatjs so, I thought if it removed from there there probably will be a different approach